frontend/src/pages/ProjectDetails.tsx

```diff
@@ -147,6 +147,7 @@ const ProjectDetailsPage = () => {
             </div>
             {project.languages.length > 10 && (
               <button
+                data-testid="show-more-languages"
```

Comment:
I believe this should be used as a last resort.
Could you try something like

`const button = screen.getByRole('button', { name: /submit/i });`

Reply:
Could you fix this so it'd be ready for the merge?